src/cljs/athens/views/athena.cljs

```diff
@@ -273,76 +229,130 @@
-                                                                                               (dispatch [:athena/toggle])
-                                                                                               (dispatch [:page/create query uid])
-                                                                                               (navigate-uid uid)))
-                                                                                 :class    (when (= i index) "selected")})
-
-                                                   [:div (use-style result-body-style)
-                                                    [:h4.title (use-sub-style result-style :title)
-                                                     [:b "Create Page: "]
-                                                     query]]
-                                                   [:span.link-leader (use-sub-style result-style :link-leader) [(r/adapt-react-class Create)]]]
-
-                                                  [:div (use-style result-style {:key      i
-                                                                                 :on-click (fn []
-                                                                                             (let [selected-page {:node/title   title
-                                                                                                                  :block/uid    uid
-                                                                                                                  :block/string string
-                                                                                                                  :query        query}]
-                                                                                               (dispatch [:athena/toggle])
-                                                                                               (dispatch [:athena/update-recent-items selected-page])
-                                                                                               (navigate-uid uid)))
-                                                                                 :class    (when (= i index) "selected")})
-                                                   [:div (use-style result-body-style)
-
-                                                    [:h4.title (use-sub-style result-style :title) (highlight-match query title)]
-                                                    (when string
-                                                      [:span.preview (use-sub-style result-style :preview) (highlight-match query string)])]
-                                                   [:span.link-leader (use-sub-style result-style :link-leader) [(r/adapt-react-class ArrowForward)]]])))]))]])))})))
+(def track-click-outside-athena
+  (partial view-utils/track-outside-click
```

Comment:
`track-outside-click` is reagent component, for it to work properly it should be invoked in `[]`, treating it is a fn is a trap.
Though seeing how you invoke `track-click-outside-athena`, maybe it works.
It would be easier to read when inlined.